packages/core/logging/core-logging-server-internal/BUILD.bazel

```diff
@@ -31,6 +31,7 @@ RUNTIME_DEPS = [
   "@npm//elastic-apm-node",
   "//packages/elastic-safer-lodash-set",
   "//packages/kbn-config-schema",
+  "//packages/kbn-std",
```

Comment:
nit: don't we need to add the types of this package in the `TYPES_DEPS` list as well?